src/code_182B30.c

```diff
@@ -175,7 +175,22 @@ INCLUDE_ASM(s32, "code_182B30", func_802597B0);
 
 INCLUDE_ASM(s32, "code_182B30", func_8025995C);
 
-INCLUDE_ASM(s32, "code_182B30", func_80259A48);
+void func_80259A48(s32 arg0, ActorPart* arg1, s32 arg2, s32 arg3) {
+    DecorationTable* decorationTable = arg1;
+
+    decorationTable = arg1->decorationTable;
+    if (decorationTable->unk_6C1 != 0) {
+        arg1->verticalStretch = 1;
+        arg1->unkOffset[0] = 0;
+        arg1->unkOffset[1] = 0;
+        decorationTable->unk_6C1 = 0;
+    }
+    if (arg0 == 0) {
```

Comment:
I think if/else-ing this would look nicer

Reply:
I don't think thats equivalent lol

Reply:
OH nvm